lib/diagnostic.js

```diff
@@ -31,7 +35,10 @@ const sum = function (previous, current) {
 // you may use any feature of JavaScript
 // to solve this problem
 const hasNone = function (array, callback) {
-
+  array.forEach((e, i, a) => {
+    if (array.indexOf(callback(e, i, a)) === -1) { return true }
+    else { return false }
```

Comment:
Close, but since this is inside a `forEach` loop it would actually return either `true` or `false` for every element in the array